packages/app/src/sandbox/eval/tests/jest-lite.js

```diff
@@ -102,12 +116,17 @@ export default class TestRunner {
       });
     };
 
+    const { JSDOM } = JSDOM2;
+    const { window } = new JSDOM('');
+    const { document } = window;
+
     return {
       ...jestTestHooks,
       expect,
       jest: jestMock,
       test,
       it,
+      document,
```

Comment:
I'm guessing this is the "globals" object. This could be pretty problematic if it's not overriding _all_ browser globals (you'd end up with `window !== document.window` in tests for example)...

Reply:
Yeah, I need to take a closer look at how Jest implements this. I initially tried to replace window, but this broke all tests for some reason. Will need to do some more testing